src/components/molecules/Wallet/Details.tsx

```diff
@@ -23,6 +27,28 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  async function addOceanToWallet() {
+    const wasAdded = await web3Provider.sendAsync({
+      method: 'metamask_watchAsset',
+      params: {
+        type: 'ERC20',
+        options: {
+          address: config.oceanTokenAddress,
+          symbol: networkName.localeCompare('Matic') === 1 ? 'mOCEAN' : 'OCEAN',
+          decimals: 18,
+          image:
+            'https://raw.githubusercontent.com/oceanprotocol/art/main/logo/token.png'
+        }
+      },
+      id: Math.round(Math.random() * 100000)
+    })
+    if (wasAdded) {
+      console.log('Added')
```

Comment:
that's nice, to make this useful for debugging we should put it behind our ocean.js `Logger.log()` and then maybe output like so:

```js
Logger.log(`Added ${tokenMetadata.params.symbol} (${tokenMetadata.params.address})`)
```

Should work easily if we move the data passed to `sendAsync()` into a const `tokenMetadata` at beginning of `addOceanToWallet()` method